### ISSUE TITLE
Firefox Upload Errors

### ISSUE THREAD

**Issue by skipjack** [dsheiko/puppetry#85]
This one [may be unrelated to puppetry][1], I just think we should track it down more to make sure it's not a issue with an outdated version on our end. Here's what the error looks like when you try to "attach a file to file input (upload)" in Firefox...

```
Error: Protocol error (DOM.setFileInputFiles): DOM.setFileInputFiles RemoteAgentError@chrome://remote/content/cdp/Error.jsm:29:5
```

![image](https://user-images.githubusercontent.com/8988697/119987471-12b6ce00-bf93-11eb-9f92-bb991d418f52.png)

[1]: https://github.com/puppeteer/puppeteer/issues/3762

**Comment by skipjack** [comment]
It seems there's a few issues related to this over in puppeteer but most are closed with PRs linked so I'm not sure what's going on.

**Comment by dsheiko** [comment]
Can you share with me the exported project?

**Comment by skipjack** [comment]
Yes, sorry for the delay. I'll work on getting that to you this week.

**Comment by skipjack** [comment]
Hey @dsheiko, check out the `firefox--upload--error.json` suite in [this repo][1]. Let me know if I can do anything else to help.

[1]: https://github.com/skipjack/puppetry-examples
